packages/react/test/specs/components/Dropdown/Dropdown-test.tsx

```diff
@@ -34,22 +34,207 @@ describe('Dropdown', () => {
     })
   })
 
+  describe('open', () => {
+    const onOpenChange = jest.fn()
+
+    afterEach(() => {
+      onOpenChange.mockReset()
+    })
+
+    it('is false when closed by trigger button click', () => {
+      const wrapper = mountWithProvider(<Dropdown onOpenChange={onOpenChange} items={items} />)
+
+      wrapper
+        .find(`button.${Dropdown.slotClassNames.triggerButton}`)
+        .simulate('click')
+        .simulate('click')
```

Comment:
nit: _Personally I_ don't like chains with `find` because they confusing:

```
wrapper
  .some
  .some
  .simulate() // it was simulated on wrapper???
```

```jsx
const wrapper = mountWithProvider(<Dropdown onOpenChange={onOpenChange} items={items} />)
const button = wrapper.find(`button.${Dropdown.slotClassNames.triggerButton}`)

button.simulate('click')
button.simulate('click')
```

Reply:
support this opinion as well - it would be better to have a dedicated variable with semantically correct name that would aid the readability:

```jsx
wrapper
  .find('.button-ok')
  .simulate() // what this button is responsible for?
```

vs 

```jsx
const confirmButton = wrapper
  .find('.button')
  
confirmButton.simulate('click')
```